Pod/Classes/URBNAlertViewController.m

```diff
@@ -16,6 +16,14 @@
 #import <URBNConvenience/URBNTextField.h>
 #import "URBNAlertAction.h"
 
+@interface URBNAlertController(Private)
```

Comment:
I’m not sure if we care about the class extension vs. category:
> A class extension bears some similarity to a category, but it can only be added to a class for which you have the source code at compile time (the class is compiled at the same time as the class extension).
[Docs](https://developer.apple.com/library/ios/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/CustomizingExistingClasses/CustomizingExistingClasses.html#//apple_ref/doc/uid/TP40011210-CH6-SW3)